samples/springmvc-sample/springmvc-consumer/src/main/java/org/apache/servicecomb/samples/springmvc/consumer/SpringmvcConsumerMain.java

```diff
@@ -51,9 +51,12 @@ public static void main(String[] args) throws Exception {
     // RestTemplate Consumer
     String sayHiResult =
         restTemplate.postForObject("cse://springmvc/springmvchello/sayhi?name=Java Chassis", null, String.class);
+    String sayHiDefaultResult =
```

Comment:
It's better to add test cases in demo projects. Sample are short examples, and demos folder are integration tests, you can add as many test cases you like. 

And maybe you need to test some primitive types default value, and from the following code, they seems not properly handled.